frontend/src/features/admin-form/responses/ResponsesPage/storage/useDecryptionWorkers.ts

```diff
@@ -2,6 +2,13 @@ import { useCallback, useEffect, useRef, useState } from 'react'
 import { useMutation, UseMutationOptions } from 'react-query'
 
 import { useAdminForm } from '~features/admin-form/common/queries'
+import {
+  trackDownloadNetworkFailure,
+  trackDownloadResponseFailure,
+  trackDownloadResponseStart,
+  trackDownloadResponseSuccess,
+  trackPartialDecryptionFailure,
+} from '~features/analytics/AnalyticsService'
```

Comment:
Just checking: have you verified these events are being reported from the worker? I'm asking because the GA base setup is now in the main app, and some events might be black-holed with the noop function 🤔

Reply:
Yup, I verified that both `trackDownloadResponseStart` and `trackDownloadResponseSuccess` are logged in GA. Wasn't sure how to test the other events though, do you have any ideas? Or can we assume that the rest will be reported too, since `trackDownloadResponseStart` and `trackDownloadResponseSuccess` are working?

Reply:
Yup, I think that's fine 👍